### DISCUSSION TITLE
Gdpr and terms and conditions

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#109] [Ideas]
I don't know what the thoughts are for T&Cs and gdpr? 
I've always tried to have a 'check boxes' to accept the t&cs all gdpr related statements, and to opt in to future marketing. These should be globally set for the organiser and then automatically applied when an event is created. 
Would love to hear others thoughts.

**Comment by daveearley** [comment]
Hi @justynpride,

Good suggestion! You can add T&Cs on he event level by creating a mandator `Radio` question with the terms added as a description.

<img width="822" alt="Screenshot 2024-07-31 at 07 27 45" src="https://github.com/user-attachments/assets/4624c878-b680-41db-bb9d-8f1735fd45a6">
<img width="774" alt="Screenshot 2024-07-31 at 07 28 10" src="https://github.com/user-attachments/assets/e8e73e0a-9c6d-4712-a45b-7a5fcce2fa2e">

There are plans to have "event defaults" which you can apply to new events. Until then, I'd suggest creating an event with the questions you'd like, and duplicating this event instead of creating events from scratch. 


<img width="1109" alt="Screenshot 2024-07-31 at 07 32 00" src="https://github.com/user-attachments/assets/e3df8027-d142-4094-9219-34f1654c458e">

**Reply by justynpride** [reply]
That makes a lot of sense and your workaround will solve the issue until you event level defaults are implemented.
